src/Stratis.Bitcoin.Features.ColdStaking/ColdStakingFeature.cs

```diff
@@ -221,6 +221,12 @@ public override void Dispose()
     /// <exception cref="InvalidOperationException">Thrown if this is not a Stratis network.</exception>
     public static class FullNodeBuilderColdStakingExtension
     {
+        public static bool RemoveSingleton<T>(this IServiceCollection services)
```

Comment:
This is more like a helper class to services maybe add a TODO to move it to such an extension.

Reply:
Done.